include/deal.II/matrix_free/hanging_nodes_internal.h

```diff
@@ -104,9 +152,6 @@ namespace CUDAWrappers
 
     namespace internal
     {
```

Comment:
Same here: I do not like the final namespace `dealii::internal::MatrixFreeFunctions::internal`. Can we skip the final `internal` namespace or rename it into something more expressive `ConstraintTypes` with shorter names inside, e.g. `type_x`, or turn this into an `enum`?